pkg/subsystem/linux/rules.go

```diff
@@ -34,7 +34,7 @@ var (
 				"syz_mount_image$exfat",
 			},
 			"fuse":     {"syz_fuse_handle_req"},
-			"gfs2":     {"syz_mount_image$gfs2", "syz_mount_image$gfs2meta"},
```

Comment:
The filesystem is still just called gfs2, also in MAINTAINERS file - what is the limitation that this has to be renamed to "cluster"?

Reply:
We currently take the subsystem name from the mailing list and the mailing list in this case is `cluster-devel@redhat.com`. But it's easy to add it to exceptions and use e.g. gfs2 as the subsystem name. Do you think it'd be better?

Reply:
Yes, I think everyone else will get confused as well because there's no filesystem named "cluster".

Reply:
Done, PTAL